Client/Frontend/AuthenticationManager/PasscodeEntryViewController.swift

```diff
@@ -25,6 +25,9 @@ class PasscodeEntryViewController: BasePasscodeViewController {
         passcodePane = PasscodePane(title: nil, passcodeSize: authInfo?.passcode?.count ?? 6)
 
         super.init()
+        
+        passcodePane.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(passcodeCheckSetup)))
+        passcodePane.isUserInteractionEnabled = true
```

Comment:
This is probably fine, but do want to add some sort of label where the keyboard is supposed to overlap that says like "Tap to enter passcode" or something (so its more obvious).

Also, if we're going to add a tap gesture to the passcode pane, we will want to make it accessible (accessible label + maybe accessibility traits?)

Reply:
I think the label is unneeded. This is a pretty weird edge case and if the user ends up in it, clicking the buttons seems obvious (to me).

The second point is definitely valid. @Brandon-T can you add an accessibility label to this Pane?

Reply:
Added accessibility to the passcode pane.